config/database.yml

```diff
@@ -24,22 +22,49 @@ defaults: &defaults
     statement_timeout: <%= IdentityConfig.store.database_statement_timeout %>
 
 development:
-  <<: *defaults
+  primary:
+    <<: *defaults
+  primary_replica:
```

Comment:
**question:** should we call it `read_replica` instead? That threw me off a little bit when you call `establish_connection`.

Reply:
Great idea, and that matches the name in devops repo where we call it `read-replica`, updated in 27926b2febc216cf1ed24280376c70bb1cada888